articles/service-bus-messaging/service-bus-samples.md

```diff
@@ -28,6 +28,11 @@ The Service Bus messaging samples demonstrate key features in [Service Bus messa
 | -------------------- | ----------------------- |
 | azure.servicebus | https://docs.microsoft.com/samples/azure/azure-sdk-for-python/servicebus-samples/ |
 
+## TypeScript samples
+| Package | Samples location | 
+| ------- | ---------------- | 
+| azure/service-bus | https://docs.microsoft.com/samples/azure/azure-sdk-for-js/service-bus-typescript/ | 
```

Comment:
Why is the `@` missing? Same for JS samples

Reply:
I've added `@` now.
I didn't add it previously because I wasn't sure of the consequences of rendering it in the `docs.microsoft.com`.


@spelluru can confirm if `@` is fine in the document.